docs/usage/request_cert.md

```diff
@@ -392,41 +107,57 @@ spec:
-| `metadata.annotations[cert.gardener.cloud/issuer]` |  Optional and may be specified if the certificate is request for a [custom domains](#Custom-Domains).  |
-| `metadata.annotations[dns.gardener.cloud/dnsnames]` | Specifies for which domains the certificate request will be created. The first entry is always taken to fill the `Common Name` field and must therefore comply with the [64 character](#Character-Restrictions) limit.  |
+### Using the custom Certificate resource
+```yaml
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
+metadata:
+  name: cert-example
+  namespace: default
+spec:
+  commonName: amazing.example.com
+  secretRef:
+    name: cert-wildcard
+    namespace: default
+  # Optionnal if using the default issuer
+  issuerRef:
+    name: garden
+```
+
+## Supported attributes
+Here is a list of all supported annotations regarding the certificate extension:
 
-### Request a wildcard certificate via Service
+|  Path  | Annotation | Value | Required | Description  |
+|:----|:----|:----|:----|:----|
+| N/A | `cert.gardener.cloud/purpose:` | `managed` | Yes when using annotations | Flag for Gardener that this specific Ingress or Service requires a certificate |
+| `spec.commonName` | `cert.gardener.cloud/commonname:` | E.g. "*.demo.example.com" or <br> "special.example.com" | Certificate and Ingress : No <br/> Service: yes | Specifies for which domain the certificate request will be created. If not specified, the names from spec.tls[].hosts are used. This entry must comply with the [64 character](#Character-Restrictions) limit. |
+| `spec.dnsName` | `cert.gardener.cloud/dnsnames:` | E.g. "special.example.com" | Certificate and Ingress : No <br/> Service: yes | Additional domains the certificate should be valid for (Subject Alternative Name). If not specified, the names from spec.tls[].hosts are used. Entries in this list can be longer than 64 characters. |
+| `spec.secretRef.name` | `cert.gardener.cloud/secretname:` | `any-name` | Yes for certificate and Service | Specifies the secret which contains the certificate/key pair. If the secret is not available yet, it'll be created automatically as soon as the certificate has been issued. |
+| `spec.issuerRef.name` | `cert.gardener.cloud/issuer:` | E.g. `gardener` | No | Specifies the issuer you want to use. Only necessary if you request certificates for [custom domains](#Custom-Domains). |
+| N/A | `cert.gardener.cloud/revoked:` | `true` otherwise always false | No | Use only to revoke a certificate, see [reference](#references) for more details |
+
+## Request a wildcard certificate
+In order to avoid the creation of multiples certificates for every single endpoints, you may want to create a wildcard certificate for your shoot's default cluster.
 
 ```yaml
-apiVersion: v1
-kind: Service
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
```

Comment:
I don't understand why you are switching to a Certificate object here.
It is convenient to still use annotations on Ingress or Service services, but always use the same wildcard domain name.

Reply:
See below